pandapower/plotting/plotly/simple_plotly.py

```diff
@@ -119,54 +119,76 @@ def simple_plotly(net, respect_switches=True, use_line_geodata=None, on_map=Fals
     OUTPUT:
         **figure** (graph_objs._figure.Figure) figure object
     """
+    node_element = "bus"
+    branch_element = "line"
+    trans_element = "trafo"
+    separator_element = "switch"
+    return _simple_plotly_generic(net, respect_switches, use_line_geodata, on_map, projection,
+                                  map_style, figsize, aspectratio, line_width, bus_size,
+                                  ext_grid_size, bus_color, line_color, trafo_color, ext_grid_color,
+                                  node_element, branch_element, trans_element, separator_element,
+                                  create_line_trace, create_bus_trace, get_hoverinfo, filename)
+
+
+def _simple_plotly_generic(net, respect_separators, use_branch_geodata, on_map, projection, map_style,
+                           figsize, aspectratio, branch_width, node_size, ext_grid_size, node_color,
+                           branch_color, trafo_color, ext_grid_color, node_element, branch_element,
+                           trans_element, separator_element, branch_trace_func, node_trace_func,
+                           hoverinfo_func, filename='temp-plot.html'):
```

Comment:
Please note here that imported functions are passed. This is because in pandapipes e.g. the function for "branch (pipes) traces" differs slightly.  I am not sure whether this is good practice or not. @friederikemeier